pkgs/servers/nextcloud/packages/apps/recognize.nix

```diff
@@ -0,0 +1,103 @@
+{
+  stdenv,
+  fetchurl,
+  lib,
+  nodejs,
+  python311,
+  util-linux,
+  ffmpeg,
+}:
+stdenv.mkDerivation rec {
+  pname = "nextcloud-app-recognise";
+  version = "8.1.1";
+
+  srcs = [
+    (fetchurl {
+      inherit version;
+      url = "https://github.com/nextcloud/recognize/releases/download/v${version}/recognize-${version}.tar.gz";
+      hash = "sha256-RIa2HzX2hfgS7KHGGBsxEdUCqDqXeAs3Xi288qA1gU4=";
+    })
+
+    (fetchurl {
+      inherit version;
+      url = "https://github.com/nextcloud/recognize/archive/refs/tags/v${version}.tar.gz";
+      hash = "sha256-op1Fdm40d8V6V+rfne0ECr60xRjBYrBPrCD3kaNeFfY=";
+    })
+    (fetchurl {
+      inherit version;
+      url = "https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-cpu-linux-x86_64-2.9.1.tar.gz";
+      hash = "sha256-f1ENJUbj214QsdEZRjaJAD1YeEKJKtPJW8pRz4KCAXM=";
+    })
```

Comment:
This is the only thing I still want to fix, to make it compatible with other platforms